plugin-server/src/worker/ingestion/property-definitions-cache.ts

```diff
@@ -34,21 +35,24 @@ export class PropertyDefinitionsCache {
         this.propertyDefinitionsCache = new Map()
     }
 
-    initialize(
-        teamId: number,
-        items: {
-            name: string
-            property_type: PropertyDefinitionsCacheValue
-        }[]
-    ): void {
+    async initialize(teamId: number, db: DB): Promise<void> {
+        const properties = await db.postgresQuery(
+            'SELECT name, property_type, type, group_type_index FROM posthog_propertydefinition WHERE team_id = $1',
```

Comment:
Should we add a limit here as best practice?

Reply:
Good question. Let's assume not for now